granian/_types.py

```diff
@@ -1,6 +1,24 @@
-from typing import Union
+from typing import Optional, Union
 
 
 class WebsocketMessage:
     kind: int
     data: Union[bytes, str]
+
+
+class HTTP1ParamType:
```

Comment:
Why do we need this, given the existence of dataclasses in https://github.com/emmett-framework/granian/blob/master/granian/http.py?